resources/assets/js/components/people/ContactSelect.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+    <div>
+        <p class="mb2" v-bind:class="{ b: required }" v-if="title">{{ title }}</p>
+        <input type="hidden" :name="name" :value="selected ? selected.id : ''">
+        <vSelect placeholder="search" @search="search" :options="results" v-model="selected"></vSelect>
+    </div>
+</template>
+
+<script>
+    import vSelect from 'vue-select'
+    import axios from 'axios'
+    export default {
+        props: {
+            name: {
+                type: String
+            },
+            title: {
+                type: String
+            },
+            required: {
+                type: Boolean
+            },
+            userContactId: {
+                type: String
+            }
+        },
+        data () {
+            return {
+                src : '/people/search',
+                results : [],
+                filterable : false,
+                selected: { label: 'Search and select an existing contact', id: '' },
+            }
+        },
+        methods: {
+            search(keyword, loading) {
+                this.getContacts(keyword, loading, this);
+            },
+            getContacts: function (keyword, loading, vm) {
+                axios.post(this.src, {
+                    needle: keyword,
+                    accountId: $('body').attr("data-account-id")
+                },{
+                    headers: {
```

Comment:
You don't need to add the headers when you are making your calls. They are added automatically, see https://github.com/monicahq/monica/blob/master/resources/assets/js/bootstrap.js#L29